composeApp/src/commonMain/kotlin/sp/bvantur/tasky/login/domain/LoginUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package sp.bvantur.tasky.login.domain
+
+import sp.bvantur.tasky.login.data.LoginRepository
+
+class LoginUseCase(private val repository: LoginRepository) {
+    suspend operator fun invoke(email: String, password: String): Boolean {
```

Comment:
It is unclear to me why you would return a boolean for this usecase. What do you think about returning the authenticated user’s details instead? If something goes wrong, you can return `null` to indicate this.

Reply:
I am trying to minimize the exposure of the data from the domain layer to the presentation layer as much as possible. So if some data is not required to be present in the presentation layer, I just don't pass it there. For now, the only information that is important with this use case is if the login was successful or not. Maybe I will add more exhaustive error handling in the future, but for the current use case, I think the Boolean type should be sufficient.